rotkehlchen/tests/unit/decoders/test_curve_lend.py

```diff
@@ -0,0 +1,694 @@
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_WETH_ARB,
+            balance=Balance(amount=FVal(receive2_amount)),
+            location_label=user_address,
+            notes=f'Receive {receive2_amount} WETH from Curve after repaying loan',
+            counterparty=CPT_CURVE,
+            address=string_to_evm_address('0x61C404B60ee9c5fB09F70F9A645DD38fE5b3A956'),
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x13cbd624Af484c4C695ff0E1d0B7e125d45F2C76']])
+def test_close_loan_using_borrowed(
+        arbitrum_one_inquirer: 'ArbitrumOneInquirer',
+        arbitrum_one_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0x34443f7d317affc6bd2c05207f1ff57b29a15039f7bdc0442feb53e279ba81b2')  # noqa: E501
+    _, underlying_token = _create_arbitrum_vault_tokens(database=arbitrum_one_inquirer.database)
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=arbitrum_one_inquirer, tx_hash=tx_hash)  # noqa: E501
+    timestamp, user_address, gas_amount, repay_amount, receive_amount, approve_amount = TimestampMS(1731958927000), arbitrum_one_accounts[0], '0.000008437093425', '0.001195651117770074', '0.450001578098145238', '115792089237316195423570985008687907853269984665640564036198.434455739459094936'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burn {gas_amount} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.PAYBACK_DEBT,
+            asset=underlying_token,
+            balance=Balance(amount=FVal(repay_amount)),
+            location_label=user_address,
+            notes=f'Repay {repay_amount} crvUSD to Curve',
+            counterparty=CPT_CURVE,
+            address=string_to_evm_address('0xB5c6082d3307088C98dA8D79991501E113e6365d'),
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_WETH_ARB,
+            balance=Balance(amount=FVal(receive_amount)),
+            location_label=user_address,
+            notes=f'Receive {receive_amount} WETH from Curve after repaying loan',
```

Comment:
Shouldn't these be withdraw? As you deposit collateral, you also withdraw collateral. Both as type/subtype and as notes description. Same as in other similar protocols.